libs/blocks/mas/mas.js

```diff
@@ -0,0 +1,32 @@
+import { loadScript, getConfig, createTag } from '../../utils/utils.js';
+
+const { miloLibs, codeRoot } = getConfig();
+const base = miloLibs || codeRoot;
+
+await loadScript(`${base}/deps/tacocat-index.js`);
+
+export default async function init(el) {
+  const offerDiv = el.querySelector(':scope > div:first-child > div');
+  const buttonDiv = el.querySelector(':scope > div:last-child > div');
+  const osi = offerDiv.textContent;
+  if (!osi) return;
```

Comment:
There is also a button only option.

Reply:
Same with the above. This is really a MVP to show folks you can do M@S on Milo.